packages/blade/src/components/Box/BaseBox/getBaseBoxStyles.test.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  getSpacingValue,
+  getBackgroundValue,
+  getBaseBoxStyles,
+  getBorderRadiusValue,
+} from './getBaseBoxStyles';
+import paymentLightTheme from '~components/BladeProvider/__tests__/paymentLightTheme';
+
+describe('getBackgroundValue', () => {
+  it('should return correct background color value', () => {
+    expect(
+      getBackgroundValue('action.background.primary.active', paymentLightTheme, 'base'),
+    ).toMatchInlineSnapshot(`"hsla(227, 100%, 45%, 1)"`);
+    expect(getBackgroundValue('red', paymentLightTheme, 'base')).toBe('red');
```

Comment:
It's on BaseBox so yup we're allowing random values + autocomplete for tokens

Reply:
https://github.com/razorpay/blade/pull/1009#discussion_r1119663353

Reply:
We should restrict it to tokens only even for internal. use, otherwise things won't be controlled purely with theme

Reply:
So our earlier Box support backgroundColor as string value only so we are kind of using string values where we get color at some other place and pass forward to Box later. Or non-background color tokens at some places.

I just didn't want to get into refactoring those things right now because these PRs are already getting huge.